File: app/src/main/java/ua/sytor/wifipass/ViewHolder.java
package ua.sytor.wifipass;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageButton;
import android.widget.LinearLayout;
import android.widget.TextView;

public class ViewHolder extends RecyclerView.ViewHolder{

    public TextView ssid;
    public TextView security;
    public TextView password;
    public LinearLayout linearLayout;
    public ImageButton imageButton;

    public ViewHolder(View itemView) {
        super(itemView);
        ssid = (TextView)itemView.findViewById(R.id.ssid);
        security = (TextView)itemView.findViewById(R.id.security);
        password = (TextView)itemView.findViewById(R.id.pass);
        linearLayout = (LinearLayout)itemView.findViewById(R.id.linearLayout);
        imageButton = (ImageButton)itemView.findViewById(R.id.dropdown);
    }
}

File: app/src/main/java/ua/sytor/wifipass/screen/wifi/ui/ViewHolder.kt
package ua.sytor.wifipass.screen.wifi.ui

import android.view.View
import android.widget.ImageButton
import android.widget.LinearLayout
import android.widget.TextView

import androidx.recyclerview.widget.RecyclerView

import ua.sytor.wifipass.R

class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    val ssidView: TextView = itemView.findViewById(R.id.ssid)
    val securityTypeView: TextView = itemView.findViewById(R.id.security)
    val passwordView: TextView = itemView.findViewById(R.id.pass)

    val linearLayout: LinearLayout = itemView.findViewById(R.id.linearLayout)
    val imageButton: ImageButton = itemView.findViewById(R.id.dropdown)

}